lib/cylc/cfgspec/suite.py

```diff
@@ -309,34 +306,27 @@ def _coerce_parameter_list(value, keys, _):
             'env-script': vdr(vtype='string'),
             'err-script': vdr(vtype='string'),
             'pre-script': vdr(vtype='string'),
-            'script': vdr(
-                vtype='string',
-                default='echo Dummy task; sleep $(cylc rnd 1 16)'),
+            'script': vdr(vtype='string'),
```

Comment:
It might be a good idea to set this to something like `echo 'no task script defined'` so that there is evidence in the job output that the script item has not been set.